kerastuner/engine/base_tuner.py

```diff
@@ -84,6 +90,11 @@ def __init__(self,
         self.hypermodel.build(hp)
         self.oracle.update_space(hp)
 
+        if reload and tf.io.gfile.exists(self._get_tuner_fname()):
+            tf.compat.v1.logging.info('Reloading Tuner from {}'.format(
```

Comment:
Why use `compat.v1.logging` here?

Reply:
Changed, thought it was necessary but switched to `tf.get_logger().info(...)`